src/Blankslate/index.tsx

```diff
@@ -0,0 +1,9 @@
+import {Blankslate as BlankslateImpl, Visual, Heading, Description, PrimaryAction, SecondaryAction} from './Blankslate'
+
+export const Blankslate = Object.assign(BlankslateImpl, {
```

Comment:
We usually combine the component and subcomponents in the `Blankslate.tsx` file? Is there a reason you moved it to the index file?

Reply:
I think it was due to trying to avoid a `default` export in `Blankslate.tsx`. Updated `Blankslate.tsx` to export a default export with the assignment 👍